From b7682ced25b82442bc658533ea7b458d8a60600f Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@st.com>
Date: Tue, 5 Apr 2022 11:26:57 +0200
Subject: [PATCH 0369/1695] dts: stm32: populate TAMP node in stm32mp251.dtsi

Add TAMP node definition under soc node in stm32mp251.dtsi.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git d61ce540f727afd68b25618f5ecd31229c40556a dts: stm32: populate TAMP node in stm32mp251.dtsi]
Signed-off-by: Gatien Chevallier <gatien.chevallier@st.com>
Change-Id: I30ee1174da0c9a2ce664a6ffc562ef063aa65201
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/248342
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/349416
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp251.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp251.dtsi b/core/arch/arm/dts/stm32mp251.dtsi
index 765ff7ac5..d8ca59605 100644
--- a/core/arch/arm/dts/stm32mp251.dtsi
+++ b/core/arch/arm/dts/stm32mp251.dtsi
@@ -596,7 +596,12 @@
 		};
 
 		tamp: tamp@46010000 {
+			compatible = "st,stm32mp25-tamp";
 			reg = <0x46010000 0x400>;
+			clocks = <&rcc CK_BUS_RTC>;
+			interrupts = <GIC_SPI 14 IRQ_TYPE_LEVEL_HIGH>;
+			#address-cells = <1>;
+			#size-cells = <1>;
 			status = "disabled";
 		};
 
-- 
2.47.2

